README.md

```diff
@@ -84,7 +100,15 @@ Define a global variable instructor and set it equal to your Squad Instructor's
 
 Your Answer:
 ```js
-// write code here
+var instructor="Matt";
+
+var localVariable=function(x){
+   console.log(x + " is now local?");
+   return x;
+};
+
+localVariable(instructor);
+
```

Comment:
:+1: 
Interesting.  I would say this satisfies the request.  You might get a kick out of checking our recommendation in the solution branch.